code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -75,8 +75,9 @@
 		//M.custom_name = created_name
 
 		brainmob.mind.transfer_to(M)
-		M.cell = locate(/obj/item/weapon/cell) in contents
-		M.cell.forceMove(M)
+		var/datum/robot_component/C = M.components["power cell"]
```

Comment:
need to dig into this later, why wouldn't this mommi's power cell component already have its wrapped value as the cell?

Reply:
changed this to add_cell function, outdated